src/mibase.ts

```diff
@@ -283,7 +283,11 @@ export class MI2DebugSession extends DebugSession {
 				response.body.threads.push(new Thread(thread.id, thread.id + ":" + threadName));
 			}
 			this.sendResponse(response);
-		}).catch(error => {
+		}).catch((error: MIError) => {
+			if (error.message === 'Selected thread is running.') {
+				this.handleMsg("stderr", error.message);
+				return;
```

Comment:
still missing some kind of response, e.g. just `this.sendResponse(response);` if you want it to return as success

Reply:
OK. I also changed it back to `console.error` because I can see it when debugging the extension and it's good enough for me.